case-repository/src/test/java/eu/itesla_project/cases/EntsoeAndXmlCaseRepositoryTest.java

```diff
@@ -137,8 +138,9 @@ public void setUp() throws Exception {
         Mockito.when(xmlImporter.exists(Matchers.isA(DataSource.class)))
                 .thenAnswer(invocation -> {
                     DataSourceMock dataSource = invocation.getArgumentAt(0, DataSourceMock.class);
-                    Path file = dataSource.getDirectory().resolve(dataSource.getBaseName() + ".xml");
-                    return Files.isRegularFile(file) && Files.exists(file);
+                    Path file_xiidm = dataSource.getDirectory().resolve(dataSource.getBaseName() + ".xiidm");
+                    Path file_xml = dataSource.getDirectory().resolve(dataSource.getBaseName() + ".xml");
+                    return ((Files.isRegularFile(file_xiidm) && Files.exists(file_xiidm)) || (Files.isRegularFile(file_xml) && Files.exists(file_xml)));
```

Comment:
Files.isRegularFile returns false if the file does not exist so the statement is useless:
return (Files.isRegularFile(file_xiidm) || (Files.isRegularFile(file_xml));